Modules/Main.lua

```diff
@@ -371,7 +375,35 @@ function main:CallMode(func, ...)
 	end
 end
 
+function main:LoadPastebinBuild()
+	if (arg[1] == nil or string.sub(arg[1], 1, 6) ~= "pob://" ) then
+		return false
+	end
+
+	local pastebinCode = string.sub(arg[1], 7, string.len(arg[1]))
+	if (string.sub(pastebinCode, -1) == "/") then
+		pastebinCode = string.sub(pastebinCode, 1, string.len(pastebinCode) - 1)
+	end
+	launch:DownloadPage("https://pastebin.com/raw/" .. pastebinCode, function(page, errMsg)
+		if errMsg then
+			self:SetMode("BUILD", false, "Failed Build Import (Download failed" .. pastebinCode .. ")")
+		else
+			local xmlText = Inflate(common.base64.decode(page:gsub("-","+"):gsub("_","/")))
+			if xmlText then
+				self:SetMode("BUILD", false, "Imported Build", xmlText)
+				self.newModeChangeToTree = true
+			else
+				self:SetMode("BUILD", false, "Failed Build Import (Decompress failed)")
+			end
+		end
+	end)
+	arg[1] = null; -- Protect against downloading again this session.
```

Comment:
Shouldn't this be `nil` instead? It doesn't cause any bugs right now, because `null` is not defined, but this looks unintentional.

Reply:
Yes, I've been away from lua too long...